app/code/VConnect/OrderVolume/Service/Order/View/Items/Columns/Volume/ChangePosition.php

```diff
@@ -0,0 +1,49 @@
+<?php
+declare(strict_types=1);
+
+namespace VConnect\OrderVolume\Service\Order\View\Items\Columns\Volume;
+
+use VConnect\OrderVolume\Model\Order\Item\Config;
+
+class ChangePosition
+{
+    private array $columnsCurrentPositionNumber = [];
+
+    public function execute(array $columns, string $afterColumnName): array
+    {
+        $this->defineColumnsCurrentPositionNumbers($columns);
+        $itemVolumeColumnCurrentPosition = $this->getColumnCurrentPositionNumber(Config::ITEM_VOLUME_COLUMN);
+        $beforeItemVolumeColumnsNumber = $this->getColumnCurrentPositionNumber($afterColumnName);
+        $beforeItemVolumeColumns = $this->arraySplice($columns, 0, $beforeItemVolumeColumnsNumber);
+        $itemVolumeColumn = $this->arraySplice($columns, $itemVolumeColumnCurrentPosition - 1, 1);
```

Comment:
You don't need create a separate private 'arraySplice' method here. use direct array_splice function to simplify the logic.